CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## [0.3.1 UNRELEASED]
+
+### Added
+
+- Added healthchecks to the [countdash-job](template/example/nomad/countdash.hcl).
```

Comment:
This should be added to 0.3.0 UNRELEASED.

You should only add a new section if there is not already a UNRELEASED one.

Reply:
Thank you. Fixed!